content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,147 @@
+
+* A [S3](https://aws.amazon.com/s3/) bucket: `s3://<S3 Bucket Name>[/<Optional Prefix>]`.
+* A [SQS](https://aws.amazon.com/sqs/) queue that passes along [SNS notifications](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/configure-sns-notifications-for-cloudtrail.html) about new log files: `sqs://<SQS Queue Name>`
+* A local filesystem path: `Any path`.
+
+The plugin also exports fields that extract information from a cloudtrail event, such as the event time, the aws region, S3 bucket/EC2 instance name, etc.
+
+The plugin uses the same authentication mechanisms used by the [AWS Go SDK](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials):
+
+* Environment Variables: specify the aws region with AWS_REGION=xxx, the access key id with AWS_ACCESS_KEY_ID=xxx, and the secret key with AWS_SECRET_ACCESS_KEY=xxx.
+* Shared Configuration Files: specify the aws region in a file at $HOME/.aws/config and the credentials in a file at $HOME/.aws/credentials.
+
+#### New Rules for AWS Cloudtrail
+
+Along with the plugin, we've created new Falco rules that look for suspicious/notable/interesting activity in cloudtrail logs. Here is an example rule:
+
+```yaml
+- rule: Console Login Without MFA
+  desc: Detect a console login without MFA.
+  condition:
+    ct.name="ConsoleLogin" and ct.error=""
+    and json.value[/userIdentity/type]!="AssumedRole" and json.value[/responseElements/ConsoleLogin]="Success"
+    and json.value[/additionalEventData/MFAUsed]="No"
+  output:
+    Detected a console login without MFA (requesting user=%ct.user, requesting IP=%ct.srcip, AWS region=%ct.region)
+  priority: CRITICAL
+  source: aws_cloudtrail
+```
+
+As the source is `aws_cloudtrail`, it is run against any events returned by the cloudtrail plugin. Also note that the rule condition uses a mix of fields from the cloudtrail plugin (`ct.name`, `ct.error`) as well as fields from the json plugin (`json.value` with json_pointer argument).
+
+Here is the full set of rule names we've written so far. The rules cover detecting any change ("Create Group", "Create Lambda Function"), detecting specific misconfigurations/bad practices ("Console Login Without MFA", "Delete Bucket Encryption"), and checking against lists of allowed/disallowed values ("Run Instances in Non-approved Region"). We would love additional rule contributions from the community!
+
+* Console Login Through Assume Role
+* Console Login Without MFA
+* Console Root Login Without MFA
+* Deactivate MFA for Root User
+* Create AWS user
+* Create Group
+* Delete Group
+* ECS Service Created
+* ECS Task Run or Started
+* Create Lambda Function
+* Update Lambda Function Code
+* Update Lambda Function Configuration
+* Run Instances
+* Run Instances in Non-approved Region
+* Delete Bucket Encryption
+* Delete Bucket Public Access Block
+* List Buckets
+* Put Bucket ACL
+* Put Bucket Policy
+* CloudTrail Trail Created
+* CloudTrail Logging Disabled
+
```

Comment:
It wold be great to have a link to these rules:

You can check these rules in our github, and use them as inspiration to contribute new ones.

Reply:
Since the PR that actually contains the rules will most likely not be merged in time for kubecon, I don't think I want to use links yet.